### DISCUSSION TITLE
Let's design a poll that to help gauge timeline urgency

### DISCUSSION THREAD

**Discussion by TelegramSam** [trustoverip/trust-spanning-protocol#40] [General]
In the 5 April WG meeting, I talked about timeline considerations () and @dhh1128 suggested we attempt a poll to understand where we are as a community in our concern for various aspects of timeline urgency. After discussion here, we'll create a poll here in github discussions. Polls have a question and 2 or more selection options.

Strawman attempt to get the conversation started:

Q: Where do you fall on the spectrum of Urgent to Completeness?

1: Urgency at all costs
2: Worth discussing, but sooner is better.
3: Need significant discussion - this isn't going to be very fast.
4: Must get this as correct as possible, even if it takes quite some time.

**Comment by talltree** [comment]
@TelegramSam I like your four choices, however you didn't include specific references to approximate time frames and example approaches (perhaps for good reason). It might be easier for folks to express preferences if those were included (along the lines of [your presentation to the TF](https://hackmd.io/@TelegramSam/HkE0a2tbn#/)). For example:

1. Urgency at all costs (e.g., small changes to an existing protocol, finish in 2-3 months).
2. Worth discussing, but sooner is better (e.g., moderate changes to an existing protocol, finish within 6 months).
3. Need significant discussion (e.g., large changes to an existing protocol, or a hybrid of multiple existing protocols, finish within a year).
4. Must get this as correct as possible (e.g., design an entirely new protocol, finish within 18-24 months).
